examples/example_compiler_gym_service/env_tests.py

```diff
@@ -27,6 +30,37 @@ def env(request) -> CompilerEnv:
         env.close()
 
 
+@pytest.fixture(
+    scope="module",
+    params=[example.EXAMPLE_CC_SERVICE_BINARY, example.EXAMPLE_PY_SERVICE_BINARY],
+)
+def bin(request) -> Path:
```

Comment:
Is this function unused?

Reply:
`@pytest.fixture` decorator declares a fixture, which means that any test function can then take `bin` as an argument and it will run and return the fixture value